tests/resources/terraform/iriusrisk-tf-aws-mapping.yaml

```diff
@@ -1,12 +1,14 @@
 trustzones:
   - id:   b61d6911-338d-46a8-9f39-8dcd24abfe91
```

Comment:
I will put on the **id** field **public-cloud** in order not to create confusion about the same UUID on both places